### DISCUSSION TITLE
Setting up Local Server (3rd step)

### DISCUSSION THREAD

**Discussion by Jelizog** [merkourisa/Augmented-Reality#5] [General]
The third step describes how to create/build a local server (with the secondary machine).

Are we supposed to make a local server? 
Or this step is the one you told us that the university will give us access to one of the Aegean's?

**Comment by merkourisa** [comment]
@Jelizog I've asked the helpdesk to install python 3 in the university labs, so you can run the local webserver. If you have a personal computer you can try setting up the environment on it
